backend/src/models/reports/index.js

```diff
@@ -5,6 +5,7 @@ const { knex, dataBaseSchemas } = dbConnection();
 // Relevant database tables
 const projectMilestoneTable = `${dataBaseSchemas().data}.project_milestone`;
 const projectTable = `${dataBaseSchemas().data}.project`;
+const portfolioTable = `${dataBaseSchemas().data}.portfolio`;
```

Comment:
I think we talked about this before, but maybe we should move away from declaring all of these at the top, and define them in the knex call itself.

Reply:
Yeah I think we still have some cleanup to do with the reports. I talked to @ASpiteri-BCGov and he said he was going to be breaking his report out into a separate file now that we have the backend folder structure worked out so I think we'll get a lot of this stuff sorted out if we move the reports to the new structure.